js/lib/components/Badge.js

```diff
@@ -25,9 +25,12 @@ export default class Badge extends Component {
     attrs.title = extract(attrs, 'label') || '';
 
     return (
-      <span {...attrs}>
-        {iconName ? icon(iconName, {className: 'Badge-icon'}) : m.trust('&nbsp;')}
-      </span>
+        <div>
+          <span {...attrs}>
+            {iconName ? icon(iconName, {className: 'Badge-icon'}) : m.trust('&nbsp;')}
+          </span>
+          <span className="Badge-label">{this.props.label}</span>
+        </div>
```

Comment:
Should this wrapping div receive some styling to prevent wrapping between the two spans?

Reply:
Also, it looks like the indentation changed? Was that on purpose? If not, please revert.

Reply:
Tthe wrapping tag should be a `span` as well because badges are inline elements.

Reply:
Do we need to return exactly one object from the `view` method or could we return multiple as well, by the way?